src/classes/LineItemTriggerTest.cls

```diff
@@ -73,6 +73,8 @@ private class LineItemTriggerTest {
 		
 		// Account manager creates accounts and orders
 		System.runAs(accountManager) {
+			List<Order__c> orders4 = [SELECT Purchased__c,Product_Category_1__c,Product_Category_Units_1__c,Product_Category_2__c,Product_Category_Units_2__c FROM Order__c];
```

Comment:
`orders4` does not match naming conventions. Please rename (hint: `myVariable`).